wftests/warrior_tests/suites/selenium_func_tests/gen_test_suite.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" ?>
+<TestSuite>
+	<Details>
+		<Name>gen_test_suite</Name>
+		<Title>selenium functional tests - generic</Title>
+		<Engineer>Sanika Kulkarni</Engineer>
+		<Date>06/21/2016</Date>
+		<Time>16:07:26</Time>
+		<type exectype="sequential_testcases"/>
+		<default_onError action="next"/>
+		<Resultsdir/>
+	</Details>
+	<Requirements/>
+	<Testcases>
+		<Testcase>
+			<path>../../testcases/selenium_func_tests/generic_tests/tc_selenium_generic_1.xml</path>
+			<context>positive</context>
+			<runtype>sequential_keywords</runtype>
+			<onError action="next"/>
+			<impact>impact</impact>
+		</Testcase>
+		<Testcase>
+			<path>../../testcases/selenium_func_tests/generic_tests/tc_selenium_generic_2.xml</path>
+			<context>positive</context>
+			<runtype>sequential_keywords</runtype>
+			<onError action="next"/>
+			<impact>impact</impact>
+		</Testcase>
+		<Testcase>
+			<path>../../testcases/selenium_func_tests/generic_tests/tc_selenium_generic_3.xml</path>
+			<context>positive</context>
+			<runtype>sequential_keywords</runtype>
+			<onError action="next"/>
+			<impact>impact</impact>
+		</Testcase>
+		<Testcase>
+			<path>../../testcases/selenium_func_tests/generic_tests/tc_selenium_generic_4.xml</path>
+			<context>positive</context>
+			<runtype>sequential_keywords</runtype>
+			<onError action="next"/>
+			<impact>impact</impact>
+		</Testcase>
+		<Testcase>
+			<path>../../testcases/selenium_func_tests/generic_tests/tc_selenium_generic_5.xml</path>
+			<context>positive</context>
+			<runtype>sequential_keywords</runtype>
+			<onError action="next"/>
+			<impact>noimpact</impact>
```

Comment:
Hi @safarheen,

Instead of marking the entire testcase as noimpact, please mark specific keywords in that testcase (which are failing due to the warning message) as noimpact. Thanks.

Reply:
Addressed the comments